versioned_docs/version-3.x/modeling/polymorphism.md

```
@@ -0,0 +1,122 @@
+model Image extends Content {
+    data Bytes
+}
+
+model Video extends Content {
+    url String
+}
+```
+
+```mermaid
+erDiagram
+	User {
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)